Examples/ChemDrawOAuthExample/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
`Create React App Sample` -> `ChemDraw Example OAuth Add-in`?

Reply:
Done